crates/daphne-server/src/roles/aggregator.rs

```diff
@@ -38,6 +40,19 @@ impl DapAggregator<DaphneAuth> for crate::App {
         let task_id_hex = task_id.to_hex();
         let durable = self.durable();
 
+        let skip_replay_protection = self
+            .kv()
+            .get_cloned::<kv::prefix::GlobalConfigOverride<bool>>(
+                &kv::prefix::GlobalOverrides::SkipReplayProtection,
+                &KvGetOptions {
+                    cache_not_found: true,
+                },
+            )
+            .await
+            .ok() // treat error as false
```

Comment:
Would it be better to abort? We should at least log the error.

Reply:
Abort could cause an incident. This is not fatal, if we can't fetch from kv:
- and we abort we create an incident
- we assume true we have a safety hole where disruptions in KV service lower our privacy guarantees
- we assume false we keep the safety guarantees even if that means crashing later anyway because we are in an incident

Logging makes sense though. Added